dev/com.ibm.ws.microprofile.config.1.4_fat/build.gradle

```diff
@@ -10,5 +10,5 @@
  *******************************************************************************/
 
 dependencies {
-  requiredLibs 'commons-logging:commons-logging:1.1.3', project(':com.ibm.ws.microprofile.config_repeat_tests'), project(':com.ibm.websphere.org.eclipse.microprofile.config.1.4')
+  requiredLibs 'commons-logging:commons-logging:1.1.3', project(':com.ibm.ws.microprofile.config_repeat_tests'), project(':com.ibm.websphere.org.eclipse.microprofile')
```

Comment:
This looks iffy. Previously we were including the 1.4 API jar, now we're including... the whole `com.ibm.websphere.org.eclipse.microprofile` project? What does that contain?

Also, it's worth noting that different versions of the MP specs often contain different conflicting versions of the same classes, so if we were putting all of them on the same classpath, that would not be ideal.

Reply:
This is depending on the project I thought.  I did not think that was adding to the classpath, but if it is then I agree that it should be changed and will look to see what the behavior is.  I thought it was just adding a dependency on the project so that it is built before this project.

Reply:
Adding things to the `requiredLibs` dependency configuration means they get packaged in the autoFVT zip lib directory so that they're on the classpath when your test runs.

Reply:
Will work on this early next week

Reply:
I put in a change to resolve this issue.  It is part of commit 1 and commit 3

Reply:
I think this problem has been solved, but also see my new comment below: https://github.com/OpenLiberty/open-liberty/pull/13265#discussion_r477366070